Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.11-slim
+
+ENV PYTHONBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
```

Comment:
Consider combining the `RUN` commands for creating directories and setting permissions into a single command to reduce the number of layers in the Docker image. For example, `RUN mkdir -p /files/media && chown -R django-user /files/media && chmod -R 755 /files/media`.